Doc/library/subprocess.rst

```diff
@@ -76,6 +76,14 @@ compatibility with older versions, see the :ref:`call-function-trio` section.
    The *universal_newlines* argument is equivalent  to *text* and is provided
    for backwards compatibility. By default, file objects are opened in binary mode.
 
+   If the current process recevies a :exc:`KeyboardInterrupt` and
+   *cleanup_timeout* is non-zero, the child process is given additional
+   time to finish before it is killed, to allow potential clean-up operations
```

Comment:
*Cleanup_timeout* is the additional duration in seconds. What objects are supported: int, float, any real number?